### PR TITLE
docs: updated Compose procedures 

### LINKED ISSUE #3791: Docs: update Compose
1. Assess docs changes related to updated Compose feature.
2. Write docs.

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

updated Compose procedures.

- Onboard the new `podman compose` command that Podman version 4.7.0 has.
- More minimalism.
- Removed the diagram as less necessary now.
- Updated Compose setup, see #4479 and #4492.

### What issues does this PR fix or reference?

fixes #3791

### CODE REVIEW

**Comment by themr0c** [comment]
@cdrage I believe that it wll play well once #4479 and #4492 are merged.

File: website/docs/compose/running-compose.md
Comment (themr0c) [inline website/docs/compose/running-compose.md]:
```suggestion  
  1. [Set the DOCKER_HOST variable](/docs/migrating-from-docker/using-the-docker_host-environment-variable).
  1. Run `docker-compose` rather than `podman compose`:
```

File: website/docs/compose/running-compose.md
Comment (themr0c) [inline website/docs/compose/running-compose.md]:
```suggestion
     ```shell-session
     $ docker-compose --file compose.yaml up --detach
     ```
```

**Comment by cdrage** [comment]
@themr0c https://github.com/containers/podman-desktop/pull/4479 and https://github.com/containers/podman-desktop/pull/4492 are merged now but just got to wait for the release and then we can showcase the steps on how to deploy / use compose.yaml

**Review by benoitf — APPROVED** [review]

**Pull request merged**
